packages/backend-data/API.md

```diff
@@ -34,6 +35,9 @@ export type DataProps = {
 // @public
 export type DataSchema = string | DerivedModelSchema;

```

Comment:
Looks like this type doesn't need to be exported anymore

Reply:
It's still in use here: https://github.com/aws-amplify/amplify-backend/pull/1213/files#diff-599f40f71e7a65474cd28950e5d5a49cf26cd7942f71796d4dd459ea251dd46fR13

Reply:
right, but it can be internal to the package, it doesn't need to be publicly exported

Reply:
Gotcha. Updated 👍🏻